lib/cloudant.js

```diff
@@ -165,6 +164,19 @@ Cloudant.prototype.create = function(model, data, options, cb) {
 };
 
 /**
+ * Create a new model instance for the given data
+ *
+ * @param {String} model The model name
+ * @param {Object} data The model data
+ * @param {Function} [cb] The cb function
+ */
+Cloudant.prototype.create = function(model, data, options, cb) {
+  debug('Cloudant.prototype.create %j %j %j ', model, data, options);
+  var self = this;
+  self._insert(model, data, cb);
```

Comment:
The original implementation ignores options while I think in the future we should take care of it, so maybe at that time we will need var self = this.

But thanks now i will change to the simple one.